pkg/gardenlet/apis/config/helper/helpers.go

```diff
@@ -91,3 +91,21 @@ func ConvertGardenletConfigurationExternal(obj runtime.Object) (*configv1alpha1.
 	}
 	return result, nil
 }
+
+// IsLoggingEnabled return true if the logging stack for clusters is enabled.
+func IsLoggingEnabled(c *config.GardenletConfiguration) bool {
+	if c != nil && c.Logging != nil &&
+		c.Logging.Enabled != nil {
+		return *c.Logging.Enabled
+	}
+	return false
+}
+
+// IsLokiEnabled return true if the loki is enabled
+func IsLokiEnabled(c *config.GardenletConfiguration) bool {
+	if c != nil && c.Logging != nil &&
```

Comment:
Can you add a test to cover the case when the passed GardenletConfiguration is nil?

Reply:
Can you please check `helpers_test.go`. Is something else required? I think those tests cover this scenario.

Reply:
I checked and I don't see a test that invokes `IsLokiEnabled` with a nil GardenletConfiguration.